ros2_batch_job/__main__.py

```diff
@@ -144,6 +145,14 @@ def main(sysargv=None):
             'image_tools_py',
             'qt_dotgraph',
         ]
+
+    # If on Windows, kill any still running `colcon` processes to avoid
+    # problems when trying to delete files from pip or the workspace during
+    # this job.
```

Comment:
Is it worth mentioning that this is a workaround for https://github.com/ros2/build_cop/issues/161 and maybe add a TODO to remove once the real issue is resolved?

Reply:
Yeah, I was gonna mention it on the that issue (rather than close it) and then make a new issue to track the removal, but I can also add a TODO.

Reply:
33ff469d4d3c6fdb43a5115708c477ba0586aecc